src/repairnator-pipeline/src/main/java/fr/inria/spirals/repairnator/process/inspectors/RepairPatch.java

```diff
@@ -201,13 +209,13 @@ private ODSLabel computeODSLabel(int patchId, Long buildId) {
 			stream.forEach(buggyLines::add);
 		} catch (IOException e) {
 			e.printStackTrace();
+			return label;
 		}
 
 		// prepare patches
 		List<String> diffLines = Arrays.asList(diff.split("\n"));
 		Patch<String> patches = UnifiedDiffUtils.parseUnifiedDiff(diffLines);
 
-		try {
 			// create a directory to store the patch: "patches/"+buildId+patchId
 			String buggyClassName = buggyFile.getName().replace(".java", "");
```

Comment:
here the code has the indentation of the previous version i.e., the block inside the removed try.